### ISSUE TITLE
clarify/clean the when a body is marked as defunct 

### ISSUE THREAD

**Issue by davewhiteland** [mysociety/fixmystreet#1248]
In fixmystreet.com cobrand we show a helpful template (`_gone_body.html`) when a body is defunct. But that conditional is currently specific to the cobrand, and tests for the absence of any areas (rather than the body being marked as deleted). Maybe this should be generalised?

We see quite a lot of installations (wrongly) set bodies up with no areas, so that alone is not enough to decide if a body is defunct; I think it may make sense to test for the body being marked as deleted. *Maybe* this means marking defunct councils in the UK FMS as deleted (which is not currently the case)?

https://github.com/mysociety/fixmystreet/blob/a428c23753694794f3b1bddace94e869c2d7550f/templates/web/base/reports/body.html#L46

Pointed out by @dracos during https://github.com/mysociety/fixmystreet/pull/1230

**Comment by dracos** [comment]
I will definitely generalise the cobrand check, it could as easily look for an existing template and use one if found. I'm confused by the no areas/deletion – currently, `/reports` lists all bodies, deleted or not, which means there is no way once a body is created of having it not appear on that page, which doesn't feel right. So presumably the no-area status/check is a way to have a body still appear in the list, but not be used for reporting (though it makes no difference currently). The checks on reporting look for bodies covering the appropriate areas, so having no areas is equivalent to being deleted.
In summary:
- [x] Change the template to look for template rather than go on cobrand moniker;
- [ ] Change /reports to not show deleted bodies?

**Comment by davewhiteland** [comment]
:+1: -- and yeah, not displaying the deleted bodies seems like the right thing to do
